src/data_transfer/data_transfer.go

```diff
@@ -17,10 +15,26 @@ const (
 	DATA_TRANSFER_HANDLER_NAME = "dataTransfer"
 )
 
+type DataTransferTopics struct {
```

Comment:
same as on ConnectionTransfer, put config in other file, store all the config in DataTransfer and get from global config